src/sign_workflow/sign_args.py

```diff
@@ -13,7 +13,7 @@
 
 class SignArgs:
     ACCEPTED_SIGNATURE_FILE_TYPES = [".sig", ".asc"]
-    ACCEPTED_PLATFORM = ["linux", "windows"]
+    ACCEPTED_PLATFORM = ["linux", "windows", "mac"]
```

Comment:
Should we say "osx" here rather than "mac"? :)

Reply:
The underlying system was using `mac` so just used that. https://github.com/opensearch-project/opensearch-signer-client/blob/main/src/opensearch-signer-client.py#L61

Reply:
Cool, cool.